src/interaction/InteractionManager.js

```diff
@@ -1327,7 +1327,11 @@ export default class InteractionManager extends EventEmitter
     {
         if (hit)
         {
-            displayObject._touchDown = true;
+            if (typeof displayObject._touchCount === 'undefined')
```

Comment:
Yeha looks like this `if` block should just be removed, mixing interaction target should set this to 0.